Unit1/Module1.5/Mini-Project1.5/README.md

```diff
@@ -0,0 +1,20 @@
+# Mini Project 1.5: Design your API Endpoints
+
+##### Credit to OpenClassrooms
+![Become](https://github.com/OCclassprojects/logo/blob/master/fav-icon.png?raw=true)
+
+### Mini Project Instructions
+
+1. Open the [project page on OpenClassrooms](https://openclassrooms.com/en/courses/6121936-build-your-web-projects-with-rest-apis/6268261-design-your-api-endpoints)
+
+2. Create a new branch and then new Pull Request and assign it to your coach for review.
+# Mini Project 1.5: Design Your API Endpoints
```

Comment:
It seems like the content is duplicated here, also I remember discussing this one. Didnt we agree, we needed to add more specific instructions for what they would actually turn in (perhaps a text file outlining their API design?)